Documentation (Code review)

AmineHLub opened the pull request:
<h1> Loom video </h1>

https://www.loom.com/share/5bd59b1d51f64a9880a61a06e3488ddd

AmineHLub committed:
Add documentation info and liscenses

AmineHLub committed:
Update live demo link

rloterh reviewed (changes requested):
Hi , 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚥 🚦  

## Good Points 👍  
- Detailed video presentation
- Live demo version is well hosted and displaying correctly
- Nice navbar
- No linter errors


## Required Changes ♻️
- [ ] Your project does not contain the `About` page. This project requires that you create 2 pages:  `Home` page and `About` page. Kindly ensure to add the `About` page
- [ ] The typography in your _hero/ top_ section does not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text, as can be seen in the provided template design. Let's ensure that the font sizes, font weights, font colors, and letter spacing matches that of the provided template design
- [ ] You have more than one `h1` tag in your code. To enhance proper use of HTML semantics let's ensure there's at least 1, but no more than 1, h1 tag on each page.
**Your hero section:**
![image](https://user-images.githubusercontent.com/12745474/132467734-73db616e-50ba-429c-aaad-58e7ffb351f5.png)

**The original hero section:**
![image](https://user-images.githubusercontent.com/12745474/132468173-c32f38a1-81d7-4148-83fe-02db1f3b5b90.png)

- [ ] The items within the _main program_ section of your design do not have appropriate padding and margins around the items, as well as spacing between the contents of each item. Kindly ensure that there's enough padding around the item's contents.
**Your program section**
![image](https://user-images.githubusercontent.com/12745474/132473652-0cf0e850-fca6-49a7-a775-9b19f6dabd7a.png)

![image](https://user-images.githubusercontent.com/12745474/132474231-2cdd31ec-e08f-4d4d-a4b0-025595eb6a03.png)

**The actual program section**
![image](https://user-images.githubusercontent.com/12745474/132472233-6338f36d-2a0f-4ba5-8b83-166013373e9f.png)

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]**._
- [ ] OPTIONAL 
Merge code of previous feature branches into the current branch so that all commits reflect in the final feature branch

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmineHLub commented:
Can you tell me which section in the design is the about page because I didn't find any so I supposed that the speakers and program sections are the about me page

Edit : Found it :+1:

AmineHLub commented:
@rloterh, Hey man I used H1 tags only four times representing each section for accessibility I mean the first one is mistaken and i'm gonna fix it with the main header but are others too ? Also can you tell me about the first header section in the main page, I removed the "Hello" title on purpose because I felt it doesn't go well with the content I'm creating. Say I go ahead and fix that and add the title what else should I implement there ? Sorry for not understanding correctly

AmineHLub committed:
Fix padding/margins in the program section

AmineHLub committed:
Fix headers margins (less space)

fix fonts weights, remove letter spacing, make paragraph bold, increase Date size, add first header h1 (hello world)

AmineHLub committed:
Style h1 for main page

rloterh commented:
> @rloterh, Hey man I used H1 tags only four times representing each section for accessibility I mean the first one is mistaken and i'm gonna fix it with the main header but are others too? Also can you tell me about the first header section in the main page, I removed the "Hello" title on purpose because I felt it doesn't go well with the content I'm creating. Say I go ahead and fix that and add the title what else should I implement there? Sorry for not understanding correctly

Please note that you need to use the `h1` tag just once in your code per page!
You can choose any text of your choice for the headings / top section, but kindly ensure that all text lines as seen in the template are replicated.

AmineHLub committed:
Replace All h1 with h2 and h3

Remove Boxshadow from mobile nav view

AmineHLub committed:
fix description paragraph font size

AmineHLub committed:
Readjust font size for date in main page

AmineHLub commented:
@rloterh Thank you for patience and your guidance. I changed all other h1 tags to h2 and h3. I have changed the paddings and the margins and the font sizes and weights for the desktop and mobile version. I don't really know if what I changed to is fine or not
![mobile](https://user-images.githubusercontent.com/83329154/132523657-6d1de2a5-d8e0-4ffc-9cc0-1b8ff3ff0e17.png)
![pc](https://user-images.githubusercontent.com/83329154/132523663-bab9b7a7-5591-4a4b-982f-92cbe1026270.png)

I'm going to create the about me page next

AmineHLub committed:
Finish About page (mobile breakpoint)

AmineHLub committed:
Add seperators

AmineHLub committed:
Finish desktop breakpoint

fix some margins and add borders and footer

AmineHLub committed:
Add links to about and index html files

AmineHLub committed:
Fix space margin between title and cards in program section

Emmyn5600 commented on index.html:
```diff
@@ -54,37 +54,38 @@
-            <h2 class="main-titling">The first online school to invest in you.</h2>
+            <span class="greetings">"Hello world!"</span>
+            <h1 class="main-titling">THE FIRST ONLINE SCHOOL TO INVEST IN YOU.</h1>
         </div>
 
         <div class="main-description">
             <p>We’re experts at training people with zero coding experience and turning them into high-earning web developer engineers. We've helped over 6000 students get hired at over 3,000 companies. Join now and become the web developper you've always wanted to be</p>
             <div class="due-date">
-                <h1>2021.10.15 (Wednesday)</h1>
-                <h2>Join before the next due date</h2>
+                <h3>2021.10.15 (Wednesday)</h3>
+                <h4>Join before the next due date</h4>
```
- [ ] The greetings `Hello world!` doesn't appear on the screen. 

- [ ] Let us include first paragraph in the card as we have in the original design and modify the font-size of our dates.

Yours

![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)

Original

![capstone](https://user-images.githubusercontent.com/55635977/132564358-5d8cd889-ff13-44aa-b686-334e08603edc.png)

- [ ] Let us align our First Heading, paragraph and h3 with dates to start on the same line, check the image below

Yours

![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)

Original

![capstone1](https://user-images.githubusercontent.com/55635977/132564363-839083e7-ccf3-4726-8a74-a6024859c2b3.png)

- [ ] Let us add the footer section. 
 
![capstone3](https://user-images.githubusercontent.com/55635977/132564372-b7f103e1-5f44-46c5-87bb-aae5924c902a.png)

AmineHLub replied:
@Emmyn5600 the changes you have requested are already there man. They probably just did not load on the live demo for some reasons but they are already. Excluding the footer which is not present but I'll add it right away

henatan99 replied:
> * [ ]  The greetings `Hello world!` doesn't appear on the screen.
> * [ ]  Let us include first paragraph in the card as we have in the original design and modify the font-size of our dates.
> 
> Yours
> 
> ![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)
> 
> Original
> 
> ![capstone](https://user-images.githubusercontent.com/55635977/132564358-5d8cd889-ff13-44aa-b686-334e08603edc.png)
> 
> * [ ]  Let us align our First Heading, paragraph and h3 with dates to start on the same line, check the image below
> 
> Yours
> 
> ![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)
> 
> Original
> 
> ![capstone1](https://user-images.githubusercontent.com/55635977/132564363-839083e7-ccf3-4726-8a74-a6024859c2b3.png)
> 
> * [ ]  Let us add the footer section.
> 
> ![capstone3](https://user-images.githubusercontent.com/55635977/132564372-b7f103e1-5f44-46c5-87bb-aae5924c902a.png)

- [ ] The alignment issue is not resolved yet. Kindly fix it. :pray: 
![image](https://user-images.githubusercontent.com/64894927/132576586-cb7990ec-3264-47db-8655-3bc369fbc7c8.png)

Emmyn5600 reviewed (changes requested):
## STATUS: CHANGES REQUESTED :recycle:

Hi, @AmineHLub 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Git/GitHub flow best practices have been followed. :heavy_check_mark:
- You have a descriptive Readme file.:heavy_check_mark:
- Your project looks nice:heavy_check_mark:
- The live link are working properly:heavy_check_mark:
- All links are working properly:heavy_check_mark:

Changes required :writing_hand: 

    * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

AmineHLub committed:
Add footer rights section

AmineHLub committed:
Fix section closing tag missing

henatan99 commented on about.html:
```diff
@@ -0,0 +1,124 @@
+                </p>
+                <a href="./index.html"><img class="logo1" src="./src/logo/main-logo.svg" alt="main-logo"></a>
+            </section>
+        </div>
```
- [ ] OPTIONAL: You have missed the See past events section on the about page. To make your project complete I strongly suggest you add it. :pray: 
![image](https://user-images.githubusercontent.com/64894927/132577414-da871cce-f94c-4065-8903-96b8dc7f731b.png)
![image](https://user-images.githubusercontent.com/64894927/132577490-ddca43a5-75d9-458b-858f-5e435b3a4d01.png)

AmineHLub replied:
Hey @henatan99, I did not forget the section in the about me I ommited it. The website is a bootcamp there are no past events to mention in there.
Now I have had 3 code reviews one of which literally asked me to change 2 out of 3 stuff that I have already changed prior to his comment and you are literally stoping the project for 10px margin which I have added on purpose just to make the design look cooler and more matching becasue the original one has an "@" in stead of that space knowing that small changes are allowed

henatan99 replied:
I am so sorry I didn't notice that it was your last review. But, the missing part was just an optional requirement though it was important. You could have populated it with something that matches your website because it is a design requirement and you should stick with it. And you also didn't make one change requested by previous reviewer. I am sorry I cannot change my review, but you can write second opinion to student success or code review leaders and they may take into consideration and change your project status if they are convinced by your second opinion. Can you do that?

AmineHLub replied:
@hentan99 it's okey it's a capstone project so I have 5 attempts. The code reviewer said : 
"The typography in your hero/ top section does not match the provided desgn template. We actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text, as can be seen in the provided template design. Let's ensure that the font sizes, font weights, font colors, and letter spacing matches that of the provided template design"
where he could just say add the first line and fix margins..
Next did not see that and then next so i thought the problem is fixed but then it wasn't..Next said change stuff I've already changed. Now with that said I thank you all for your cooperation and time. In anyways I fixed the issue with the margin that you have requested for and I sent another code review attempt

henatan99 reviewed (changes requested):
Hi @AmineHLub , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmineHLub committed:
Fix margin on join before due date

emmanuellekamwa reviewed (approved):
**STATUS: APPROVED ✔🎉**
Hello @AmineHLub 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it. 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
